apps/formbricks-com/components/home/Steps.tsx

```diff
@@ -21,9 +21,10 @@ export const Steps: React.FC = () => {
         heading="Set Formbricks up in minutes"
         subheading="Formbricks is designed for as little dev attention as possible. Here’s how:"
       />
-      <div id="howitworks" className="mx-auto mb-12 mt-16 max-w-lg md:mb-0 md:mt-8 md:max-w-none">
+      <div id="howitworks" className="m-auto mb-12 mt-16 max-w-lg md:mb-0 md:mt-8 md:max-w-none">
         <div className="px-4 sm:max-w-4xl sm:px-6 lg:max-w-7xl lg:px-8">
           <div className="grid md:grid-cols-2 md:items-center md:gap-16">
+            {/* <div className="inline-flex flex-col items-center"> */}
```

Comment:
small nit: Can you remove this additonal commented line please?

Reply:
Yeah! Sure. 
Just removing right now.